controllers/controllers/resource/template.go

```diff
@@ -72,11 +72,11 @@ func (r *VsphereTemplate) TemplateResources(ctx context.Context, eksaCluster *an
 }
 
 func generateTemplateResources(builder providers.TemplateBuilder, clusterSpec *cluster.Spec, buildOptions ...providers.BuildMapOption) ([]*unstructured.Unstructured, error) {
-	cp, err := builder.GenerateClusterApiSpecCP(clusterSpec, buildOptions...)
+	cp, err := builder.GenerateCapiSpecCP(clusterSpec, buildOptions...)
 	if err != nil {
 		return nil, err
 	}
-	md, err := builder.GenerateClusterApiSpecMD(clusterSpec, buildOptions...)
+	md, err := builder.GenerateCapiSpecMD(clusterSpec, buildOptions...)
```

Comment:
nit: I think it should be `CAPI` instead of `Capi`

Reply:
i can make the change across the board